src/Elastic.Transport/Diagnostics/OpenTelemetry/OpenTelemetry.cs

```diff
@@ -51,20 +53,7 @@ internal static void SetCommonAttributes(Activity? activity, OpenTelemetryData o
 		}
 
 		var productSchemaVersion = string.Empty;
-		if (openTelemetryData.SpanAttributes is not null)
-		{
-			foreach (var attribute in openTelemetryData.SpanAttributes)
-			{
-				activity?.SetTag(attribute.Key, attribute.Value);
-
-				if (attribute.Key.Equals(OpenTelemetryAttributes.DbElasticsearchSchemaUrl, StringComparison.Ordinal))
-				{
-					if (attribute.Value is string schemaVersion)
-						productSchemaVersion = schemaVersion;
```

Comment:
@stevejgordon Removing this causes the condition in line 69 to be always `true`.